dev-support/test-patch.sh

```diff
@@ -2389,10 +2214,11 @@ function precheck_without_patch
 {
   local result=0
 
-  precheck_mvninstall
-
-  if [[ $? -gt 0 ]]; then
-    ((result = result +1 ))
+  if declare -f ${BUILDTOOL}_precheck_install >/dev/null; then
+    "${BUILDTOOL}_precheck_install"
+    if [[ $? -gt 0 ]]; then
+      ((result = result +1 ))
```

Comment:
shellcheck: 5: note: Double quote to prevent globbing and word splitting. [SC2086]